Buildings/Experimental/DHC/EnergyTransferStations/Cooling/Indirect.mo

```diff
@@ -0,0 +1,301 @@
+model Indirect
+  "Indirect cooling energy transfer station for district energy systems"
+  extends
+    Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.PartialETS(
+    QChiWat_flow_nominal=-Q_flow_nominal,
+    final typ=DHC.Types.DistrictSystemType.Cooling,
+    final have_weaBus=false,
+    final have_chiWat=true,
+    final have_heaWat=false,
+    final have_hotWat=false,
+    final have_eleHea=false,
+    final nFue=0,
+    final have_eleCoo=false,
+    final have_pum=false,
+    final have_fan=false,
+    nPorts_aChiWat=1,
+    nPorts_bChiWat=1);
+  // mass flow rates
+  parameter Modelica.Units.SI.MassFlowRate mDis_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of district cooling side"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mBui_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of building cooling side"
+    annotation(Dialog(group="Nominal condition"));
+  // Primary supply control valve
+  parameter Modelica.Units.SI.PressureDifference dpConVal_nominal(
+    final min=0,
+    displayUnit="Pa")=6000
+    "Nominal pressure drop of fully open control valve"
+    annotation(Dialog(group="Nominal condition"));
+  // Heat exchanger
+  parameter Modelica.Units.SI.PressureDifference dp1_nominal(
+    final min=0,
+    final start=500,
+    displayUnit="Pa")
+    "Nominal pressure difference on primary side"
+    annotation (Dialog(group="Heat exchanger"));
+  parameter Modelica.Units.SI.PressureDifference dp2_nominal(
+    final min=0,
+    final start=500,
+    displayUnit="Pa")
+    "Nominal pressure difference on secondary side"
+    annotation (Dialog(group="Heat exchanger"));
+  parameter Boolean use_Q_flow_nominal=true
+    "Set to true to specify Q_flow_nominal and temeratures, or to false to specify effectiveness"
+    annotation (Dialog(group="Heat exchanger"));
+  parameter Modelica.Units.SI.HeatFlowRate Q_flow_nominal(
+    final max=0,
+    final start=-100000)
+    "Nominal heat transfer"
+    annotation (Dialog(group="Heat exchanger"));
+  parameter Modelica.Units.SI.Temperature T_a1_nominal(
+    final min=0+273.15,
+    final max=100+273.15,
+    final start=5+273.15,
+    displayUnit="K")
+    "Nominal temperature at port a1"
+    annotation (Dialog(group="Heat exchanger"));
+  parameter Modelica.Units.SI.Temperature T_a2_nominal(
+    final min=0+273.15,
+    final max=100+273.15,
+    final start=7+273.15,
+    displayUnit="K")
+    "Nominal temperature at port a2"
+    annotation (Dialog(group="Heat exchanger"));
+  parameter Modelica.Units.SI.Efficiency eta(
+    final min=0,
+    final max=1)=0.8
+    "Constant effectiveness"
+    annotation (Dialog(group="Heat exchanger"));
+   //Controller parameters
+  parameter Real k(
+    final min=0,
+    final unit="1")=1
+    "Gain of controller";
+  parameter Modelica.Units.SI.Time Ti(
+    final min=Modelica.Constants.small)=120
```

Comment:
`Ti` and `Td` could be disabled depending on the controller type.

Reply:
Agree, if the controller type is not PID control, they will be redundant.